datafusion/sqllogictest/test_files/window.slt

```diff
@@ -4159,15 +4176,34 @@ NULL def NULL def
 NULL def x x
 x x NULL NULL
 
+# LAG window function IGNORE/RESPECT NULLS support with descending order and nondefault offset trigger evaluate_all
```

Comment:
I think, this test is exactly same version of the test below. If so, it is better to remove one of them.

Reply:
Sure, I'll remove it